en/deploy/README.md

```diff
@@ -167,7 +167,7 @@ Installing setuptools, pip...done.
 
 $ source myvenv/bin/activate
 
-(myvenv) $  pip install django==1.9
+(myvenv) $  pip install "django==1.9.*"
```

Comment:
I would also opt on the setting proposed by @alexanderad. Using * in pip packages is not very popular and I actually had to check if something like that works... :)